app/controllers/application.js

```diff
@@ -24,8 +25,18 @@ export default Ember.Controller.extend({
       // jshint ignore:line
       this.redirectToItem();
     }
+
+    this.get("subscription").on("change:message", this, this.onNewNotification);
   }),
 
+  onNewNotification(notification) {
+    const msg = this.get("store").peekRecord("message", notification.record.id);
+
+    if (msg.get("isUnread")) {
+      this.get("messagesUtil")._incrementCount();
```

Comment:
`_incrementCount` shouldn't be called from outside the service. It's meant as a private method (thus the leading `_`)